src/views/Wizard/views/DataplaneUniversal.vue

```diff
@@ -594,64 +592,15 @@ export default {
       // const cpAddress = this.$store.getters.getConfig.general.advertisedHostname
       const { univDataplaneId } = this.validate
       const cmdStructure = `kuma-dp run \\
-      --cp-address=${this.getDpServerUrl} \\
+      --cp-address=${kumaDpServerUrl()} \\
       --dataplane=${`"${json2yaml(this.getDataplaneSchema)}"`} \\
       --dataplane-token-file=kuma-token-${univDataplaneId}`
 
       return cmdStructure
-    }
-  },
-  watch: {
-    validate: {
-      handler () {
-        const data = JSON.stringify(this.validate)
-        const mesh = this.validate.meshName
-
-        const {
-          univDataplaneServiceName,
-          univDataplaneId,
-          univDataplaneNetworkAddress,
-          univDataplaneNetworkServicePort,
-          univDataplaneNetworkDPPort,
-          univDataplaneNetworkProtocol
-        } = this.validate
-
-        // write the v-model data to localStorage whenever it changes
-        localStorage.setItem('storedFormData', data)
-
-        // allow the user to proceed if they've selected a Mesh
-        mesh.length
-          ? this.nextDisabled = false
-          : this.nextDisabled = true
-
-        // networking field validation
-        if (this.$route.query.step === 2) {
-          if (
-            univDataplaneNetworkAddress &&
-            univDataplaneNetworkServicePort &&
-            univDataplaneNetworkDPPort &&
-            univDataplaneNetworkProtocol
-          ) {
-            this.nextDisabled = false
-          } else {
-            this.nextDisabled = true
-          }
-        }
-
-        // topology field validation
-        if (this.$route.query.step === 1) {
-          if (univDataplaneServiceName && univDataplaneId) {
-            this.nextDisabled = false
-          } else {
-            this.nextDisabled = true
-          }
-        }
-      },
-      deep: true
     },
 
-    '$route' () {
-      const step = this.$route.query.step
+    nextDisabled() {
+      const mesh = this.validate.meshName
 
       const {
         univDataplaneServiceName,
```

Comment:
You can destructurize the `.meshName` and change the name to `.mesh` the same as in one of the suggestions before I think